mne/io/kit/tests/test_kit.py

```diff
@@ -185,4 +188,24 @@ def test_hsp_elp():
     assert_array_almost_equal(pts_elp_in_dev, pts_txt_in_dev, decimal=5)
 
 
+def test_decimate():
+    """Test decimation of digitizer headshapes with too many points. """
+    # load headshape and convert to meters
+    hsp = np.loadtxt(sphere_hsp_txt_path)
+    hsp /= 1000
```

Comment:
I used `NamedTemporaryFile` from the `tempfile` module before seeing this comment. Would using `_TempDir()` be better?